huntsman/camera/pyro.py

```diff
@@ -436,87 +406,20 @@ def __init__(self, config_files=None):
         module = load_module('pocs.camera.{}'.format(camera_config['model']))
         self._camera = module.Camera(**camera_config)
 
-# Properties
-
-    @property
-    def name(self):
-        return self._camera.name
-
-    @property
-    def model(self):
-        return self._camera.model
-
-    @property
-    def uid(self):
-        return self._camera.uid
+# Properties - rather than labouriously wrapping every camera property individually expose
+# them all with generic get and set methods.
 
-    @property
-    def readout_time(self):
-        return self._camera.readout_time
+    def get(self, property_name, subcomponent=None):
+        obj = self._camera
+        if subcomponent:
+            obj = getattr(obj, subcomponent)
+        return getattr(obj, property_name)
```

Comment:
I like the cleanup on this side of things but am a little worried about the string based calling from the user's perspective.  See note in review comments.